graphql_api/types/owner/owner.graphql

```diff
@@ -23,6 +23,7 @@ type Owner {
   orgUploadToken: String
   defaultOrgUsername: String
   isCurrentUserActivated: Boolean!
+  tier: TierName
```

Comment:
The return type of the resolver is `TierName` which indicates that it can't be null.  Should this be `TierName!` to match?  Or should the return type of the resolver be `Optional[TierName]`?

Reply:
Good catch, I'll make this non-nullable